src/backend/sgx/attestation.rs

```diff
@@ -16,7 +16,7 @@ use std::slice::{from_raw_parts, from_raw_parts_mut};
 use protobuf::Message;
 use sallyport::syscall::{SGX_QUOTE_SIZE, SGX_TI_SIZE};
 
-const AESM_SOCKET: &str = "/var/run/aesmd/aesm.socket";
+pub const AESM_SOCKET: &str = "/var/run/aesmd/aesm.socket";
```

Comment:
This should move up one level to `backend::sgx`.

Reply:
Thanks for the feedback. I will add all the changes and fix the checks once the server is back up :)